edrop-gunicorn.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+NAME="edrop"                                  # Name of the application
+DJANGODIR=/edrop                              # Django project directory
+NUM_WORKERS=3                                 # Number of Gunicorn workers
+DJANGO_SETTINGS_MODULE=edrop.settings         # Django settings module
+DJANGO_WSGI_MODULE=edrop.wsgi                 # WSGI module name
+
+echo "Starting $NAME as `whoami`"
+
+# Activate the virtual environment
+cd $DJANGODIR
+source .env_app
+export DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
+export PYTHONPATH=$DJANGODIR:$PYTHONPATH
+mkdir -p /edrop/logs
+python -m pip install -r requirements.txt
+python manage.py migrate
+python manage.py collectstatic --noinput
+service supervisor start
```

Comment:
this should not be needed here (and would actually lead to an infinite loop of starting supervisor (or error out maybe), since this is the script that supervisor runs to start gunicorn.